### PR TITLE
Remove the UserList component from everywhere

### PR DESCRIPTION
**Pull request by mkitt**
The UserProfile will replace it and there is no more list view for user collections.

I think I get them all. I hunted with `usersAsList`, `UserList`. Might be a few stragglers out there?

:knife::fire:

### CODE REVIEW

File: src/reducers/gui.js [hunk 1]
```diff
@@ -41,8 +41,17 @@ const NO_LAYOUT_TOOLS = [
   /^\/notifications\b/,
   /^\/settings\b/,
   /^\/onboarding\b/,
+  /^\/[\w\-]+\/following\b/,
+  /^\/[\w\-]+\/followers\b/,
 ]
```
Comment (mkitt) [inline src/reducers/gui.js]: I think those are good. 💉

File: src/reducers/gui.js [hunk 2]
```diff
@@ -41,8 +41,17 @@ const NO_LAYOUT_TOOLS = [
...
 
+function getIsLayoutToolHidden() {
+  const pathname = location.pathname
+  const locationType = get(location, 'query.type')
+  const isUserSearch = locationType === 'users' && /^\/search\b/.test(pathname)
+  return isUserSearch || NO_LAYOUT_TOOLS.some(pagex => pagex.test(pathname))
+}
+
```
Comment (mkitt) [inline src/reducers/gui.js]: I'm not sure if there is a better way to do this here. It's sort of a weird edge case where we can't just use search since Search results are still grid/list mode.

Comment (rynbyjn) [inline src/reducers/gui.js]: This seems reasonable for now.
